clouds/snowflake/modules/sql/quadbin/QUADBIN_BBOX.sql

```diff
@@ -1,34 +1,50 @@
 ----------------------------
--- Copyright (C) 2022 CARTO
+-- Copyright (C) 2023 CARTO
```

Comment:
The dates should be 2022-2023 as we do in the branch `chore/sc-320182/optimize-quadbin-polyfill-snowflake`